db/seeds.rb

```diff
@@ -24,5 +24,13 @@
   else
     flag = false
   end
-  patient.pregnancies.create({last_menstrual_period_time: DateTime.new(2016,1,1), urgent_flag: flag})
+  pregnancy = patient.pregnancies.create({last_menstrual_period_time: DateTime.new(2016,1,1), urgent_flag: flag})
+  5.times do
+    pregnancy.calls.create({status: 'Left voicemail', creating_user_id: user.id})
+  end
+  if patient.name == 'Patient 0'
+    10.times do
+      pregnancy.calls.create({status: 'Reached patient', creating_user_id: user.id})
```

Comment:
Ah ok, yeah that did the trick. It looks like the `Call` model was updated after my fork was made, any way we could add an automated test for `rake db:seed` so that CI will catch it when this happens?